app/views/councilTaxUpload.scala.html

```diff
@@ -47,8 +54,9 @@
     @initiateResponse.map { ir =>
         <input type="hidden" id="councilTaxUploadEmptyFileError" value="@(routes.CouncilTaxUploadController.onPageLoad(true))" />
         <input type="hidden" id="councilTaxUploadReportError" value="@(routes.CouncilTaxUploadController.onError(ir.reference))" />
-        <input type="hidden" id="councilTaxUploadFormRedirect" value="@(routes.ConfirmationController.onPageLoad(ir.reference))" />
+        <input type="hidden" id="councilTaxUploadFormRedirect" value="@(routes.ConfirmationController.onPageRefresh(ir.reference))" />
```

Comment:
When redirecting - using the `pageRefresh` which is similar to onPageLoad, except it return the status as well... - Might be able to delete onPageLoad, but I leave it for some tech debt